src/main/resources/network/brightspots/rcv/hints_cvr_files.txt

```diff
@@ -4,7 +4,7 @@ The tabulator needs to know where each of your CVR files is and how to interpret
 
 For each of your CVR files, provide the necessary information and then use the Add button to add it to the list.
 
-Provider (required): The vendor/machine that generated (produced) the file. After you select the field, the tabulator will fill in as many of the other fields as it can based on what it knows about that provider. You can adjust those values as necessary. Additional information about supported formats is available at https://github.com/BrightSpots/rcv/wiki/CVR-Formats.
+Provider (required): The vendor/machine that generated (produced) the file. After you select the field, the tabulator will fill in as many of the other fields as it can based on what it knows about that provider. You can adjust those values as necessary. Additional information about supported formats is available in our documentation.
```

Comment:
Just a heads-up that "in our documentation" will likely be confusing (and possibly frustrating) to users.